### PR TITLE
64x64 Sprites Conversion

### PR DESCRIPTION
**Pull request by TheDZD**
Yeah no that's not happening yet.

### CODE REVIEW

**Comment by KasparoVy** [comment]
lmfao, all the resprites that would have to be done

rip rest of summer

**Comment by TheDZD** [comment]
Yup, and there's a good chance it might make icon procs significantly (up to 4x) more expensive, and considering how horribly expensive our human update_icons() already is, it would become *the* most expensive proc bar none.

File: html/changelog.html [hunk 1]
```diff
@@ -55,6 +55,74 @@
...
+			<h2 class="date">01 August 2016</h2>
+			<h3 class="author">Crazylemon64 updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">Admins are now able to print a map of the z levels.</li>
+				<li class="rscadd">Admins are now capable of scrambling the z levels' transitions.</li>
+				<li class="bugfix">Janiborg trash bags no longer fit down disposals when empty</li>
+			</ul>
+			<h3 class="author">Fethas updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="bugfix">(Maybe?)..We actually check if the race HAS BLOOD before trying to suck them</li>
```
Comment (LittleBigKid2000) [inline html/changelog.html]: lewd

**Comment by Lady-Luck** [comment]
waifu2x would provide a great baseline for a 64x64 set

**Pull request merged**